src/filesink.cpp

```diff
@@ -54,12 +54,12 @@ namespace g3 {
       out << message.get().toString() << std::flush;
    }
 
-   std::string FileSink::changeLogFile(const std::string &directory) {
+   std::string FileSink::changeLogFile(const std::string &directory, const std::string &logger_id) {
```

Comment:
I added the arg, where the call to createLogFileName was called in the code,
I want to be able to specify my own moniker. Hence I need to be able to provide this somewhere in the public interface...